tf_encrypted/protocol/protocol.py

```diff
@@ -6,6 +6,7 @@
 
 import tensorflow as tf
 
+import tf_encrypted as tfe
```

Comment:
not a big fan of the mixing importing style. i think we should rewrite the way this works instead tbh.

Reply:
Yeah I'm not sure the best way to handle this.